apps/web/.env.template

```
@@ -7,8 +7,16 @@ NEXT_PUBLIC_TWITTER_HANDLE=@ecency_official
 NEXT_PUBLIC_NWS_SERVER=wss://enotify.ecency.com
-NEXT_PUBLIC_GMAPS_API_KEY=
-NEXT_PUBLIC_GMAPS_MAP_ID=
+# Geo-tag picker (all optional). Search uses an offline cities dataset + an
+# open-source geocoder; the map uses OpenStreetMap tiles. The public defaults
+# below are fine for normal use — requests are made client-side (distributed
+# across users' browsers), debounced, and most lookups hit the offline dataset.
+# Only override these to point at a self-hosted geocoder / tile provider if
+# volume ever grows enough to warrant it. NOTE: if you change these hosts, also
+# add their origins to the CSP connect-src / img-src in next.config.js (the
+# defaults are derived automatically from these vars).
+NEXT_PUBLIC_GEOCODER_HOST=https://photon.komoot.io
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_GEOCODER_HOST key should go before the NEXT_PUBLIC_IMAGE_SERVER key

(UnorderedKey)